packages/react/src/analytics/integrations/Vitorino/Vitorino.ts

```diff
@@ -10,59 +10,54 @@
  * ```
  */
```

Comment:
The tsdoc for this class should be updated as well.

Reply:
Fixed